### DISCUSSION TITLE
Shrink brand icons

### DISCUSSION THREAD

**Discussion by AJediIAm** [home-assistant/frontend#17918] [Theming]
The brand icons are very large compared to the other icons on a dashboard. 
They look out of place:
![image](https://github.com/home-assistant/frontend/assets/5879533/176c0d72-51ed-469d-9ee3-a75e24c67eea)

I managed to hack the HTML to make the icons a bit smaller (although they lost their roundness in the process)
![image](https://github.com/home-assistant/frontend/assets/5879533/a27f1a74-ef76-4ae9-b817-7a8f4c39f329)

I think the smaller icons look much better, but unfortunately, I do not have the skills to convert the hacking into in pull request.
![image](https://github.com/home-assistant/frontend/assets/5879533/fa3c6393-f2a5-44c5-9024-089fa5630719)

_Therefore, I call upon the mighty web wizard, to wield their mystical powers and transmute the grand icons into the mighty yet minuscule form fitting for it stature in the HA-realm. Let their dimensions befit their significance, and in doing so, elevate the essence of our user interface to new heights._

**Comment by ildar170975** [comment]
How do you paste these brand icons in a Entities card's code?

**Reply by AJediIAm** [reply]
I'm not sure if I understand the question. 
I have not changed the icon. I just modified the css which defines how the icon is displayed. Specifically the background-size, background-repeat and background-position.

**Reply by ildar170975** [reply]
I meant a diff. thing.

The screenshot you posted:
![267741885-a27f1a74-ef76-4ae9-b817-7a8f4c39f329](https://github.com/home-assistant/frontend/assets/71872483/30251963-bf0e-49d8-a825-f79fcf2a4ef5)
- is it a part of some HA settings in Devices pages,
- or is it a user-defined Entities card?

If the latter - then I am asking how did you add a row with this brand icon?

**Comment by AJediIAm** [comment]
Its the standard devices page.
